npm/qsharp/generate_katas_content.js

```diff
@@ -649,10 +672,16 @@ function generateKatasContent(katasPath, outputPath) {
     indexPath,
     "Could not read the contents of the katas index file",
   );
-  const katasDirs = tryParseJSON(
-    indexJson,
-    `Invalid katas index at ${indexPath}`,
+  const publishedKatasDirs = Object.values(
+    tryParseJSON(indexJson, `Invalid katas index at ${indexPath}`),
   );
```

Comment:
Unless I'm missing something, I don't think `Object.values()` is needed here? `index.json` seems to declare a string array, so the type of `publishedKatasDirs` should just be a string array without you having to do anything.

Reply:
You're right. Odd, at one point I couldn't make it work without that... Apparently, I changed something afterwards :-)